### DISCUSSION TITLE
Add Option for React Syntax

### DISCUSSION THREAD

**Discussion by thecodeclimber** [alpinejs/alpine#661] [4. General]
The one thing I do not like about vue syntax is the quotation marks for v-on and the like.  I much prefer the react syntax of squirly brackets.  In other words:

```
// vue syntax
<div @click="isOpen = !isOpen">...</div>

// react syntax
<div @click={isOpen = !isOpen}>...<div>
```

The squirly brackets help indicate that this is some sort of logic as opposed to a normal html attribute.  I imagine it is not possible to switch out the syntax, but perhaps you could add the option to use both options.

**Comment by SimoTod** [comment]
The curly bracket thing in react is not a style convention but it's the way JSX supports interpolation.
Alpine doesn't use a virtual dom, what you write goes straight into the page and if you don't put quotes, browsers won't understand the source properly so it's technically not feasible.

**Comment by thecodeclimber** [comment]
Okay - got it.  I'll just learn to live with it :).

Thanks for the quick reply.
